shell/renderer/electron_renderer_client.cc

```diff
@@ -85,8 +85,8 @@ void ElectronRendererClient::DidCreateScriptContext(
     node::tracing::TraceEventHelper::SetAgent(node::CreateAgent());
 
   // Setup node environment for each window.
-  bool initialized = node::InitializeContext(renderer_context);
-  CHECK(initialized);
+  v8::Maybe<bool> initialized = node::InitializeContext(renderer_context);
+  CHECK(!initialized.IsNothing());
```

Comment:
Do we also need to CHECK that the value of the bool inside the maybe is `true`?

Reply:
oh good catch, thanks!